frontend/apps/reader/modules/readerscrolling.lua

```diff
@@ -162,6 +162,7 @@ Default value: %1 ms]]), scroll_activation_delay_default_ms),
                         value_max = 2000,
                         value_step = 100,
                         value_hold_step = 500,
+                        unit = _("ms"),
```

Comment:
For ms and s a context isn't necessary exactly, but it can be useful to filter by context and in this instance as built-in documentation.